File: overview/rev1/guides/examples/java/reversal-create.java
// creditHref is the stored href for the credit
Credit credit = new Credit(creditHref);

HashMap<String, Object> meta = new HashMap<String, Object>();
meta.put("user.refund_reason", "not happy with product");
meta.put("merchant.feedback", "positive");
meta.put("fulfillment.item.condition", "OK");

HashMap<String, Object> payload = new HashMap<String, Object>();
payload.put("amount", 100000);
payload.put("description", "Reversal for Order #1111");
payload.put("meta", meta);

try {
    credit.reverse(payload);
}
catch (HTTPError e) {}

File: overview/rev1/guides/examples/csharp/reversal-create.cs
// credit_href is the stored href for the Credit
Credit credit = Credit.Fetch(credit_href);
Dictionary<string, string> payload = new Dictionary<string, string>();
payload.Add("amount", "3000");
payload.Add("description", "Reversal for Order #1111");
Reversal reversal = credit.Reverse(payload);